### PR TITLE
Updated lunch

### PR DESCRIPTION
**Pull request by asyasky**
Update lunch options for 2022 and uncomment room fields.

### CODE REVIEW

File: ServerCore/Pages/Teams/Create.cshtml [hunk 1]
```diff
@@ -34,13 +34,12 @@ else
                 <input asp-for="Team.Name" class="form-control" />
                 <span asp-validation-for="Team.Name" class="text-danger"></span>
             </div>
-            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt)
-    <div class="form-group">
+            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt) -->
```
Comment (morganbr) [inline ServerCore/Pages/Teams/Create.cshtml:37]: Should this be removed?

File: ServerCore/Pages/Teams/Details.cshtml [hunk 2]
```diff
@@ -114,7 +114,7 @@
         @Html.DisplayFor(model => model.Team.RoomID)
     </dd>
     -->
-                <!--
+                
                 <dt>
                     Team room
                 </dt>
```
Comment (morganbr) [inline ServerCore/Pages/Teams/Details.cshtml:120]: Line 130 doesn't match the create/edit pages -- it asks for a room, but the other pages wouldn't allow it

File: ServerCore/Pages/Teams/Edit.cshtml [hunk 3]
```diff
@@ -26,29 +26,26 @@
...
                 <span asp-validation-for="Team.Name" class="text-danger"></span>
             </div>
-            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt)
-    <div class="form-group">
-        <label asp-for="Team.RoomID" class="control-label"></label>
-        <input asp-for="Team.RoomID" class="form-control" />
-        <span asp-validation-for="Team.RoomID" class="text-danger"></span>
-    </div>
-    -->
+            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt) -->
```
Comment (morganbr) [inline ServerCore/Pages/Teams/Edit.cshtml:29]: Should this be removed?

**Review by morganbr — APPROVED** [review]

File: ServerCore/Pages/Teams/Create.cshtml (hunk 1 above)
Comment (asyasky) [reply ServerCore/Pages/Teams/Create.cshtml:37]: I'm not sure what the extra (unconditional) room label is and I think that that's what this applies to, so I want to leave it for the moment.

File: ServerCore/Pages/Teams/Details.cshtml (hunk 2 above)
Comment (asyasky) [reply ServerCore/Pages/Teams/Details.cshtml:120]: It's allowed on create/edit if it's not a intern event (that's what the if/else is for) or if it's on the admin view for the intern event.

**Pull request merged**